benchmarks/serialize.py

```diff
@@ -198,9 +198,9 @@ def build_parser() -> argparse.ArgumentParser:
     )
     param_groups.add_argument(
         "--shard",
-        help="shard size (default: 1000000)",
+        help="shard size (default: 1073741824)",
         type=int,
-        default=1000000,
+        default=1_073_741_824,
```

Comment:
applies everywhere: 
any thoughts on gigabyte vs gibibyte (to use IEC 80000-13 terms)
1_000_000_000 vs 1_073_741_824?

Reply:
Either works, up to you if you want to change

Reply:
no clear winner in benchmarking, leaving as is

Reply:
Although looking at the shard definitions/goldens, probably easier to reason about `1_000_000_000`